src/spec-node/devContainersSpecCLI.ts

```diff
@@ -974,9 +965,6 @@ function readConfigurationOptions(y: Argv) {
 			if (idLabels?.some(idLabel => !/.+=.+/.test(idLabel))) {
 				throw new Error('Unmatched argument format: id-label must match <name>=<value>');
 			}
-			if (!argv['container-id'] && !idLabels?.length && !argv['workspace-folder']) {
```

Comment:
Same as with `run-user-commands`, `read-configuration` can be run without workspace folder.